content/tutorials/manage-the-rippled-server/installation/install-rippled-on-ubuntu.md

```diff
@@ -31,23 +31,22 @@ Before you install `rippled`, you must meet the [System Requirements](system-req
 
         sudo mkdir /usr/local/share/keyrings/
         wget -q -O - "https://repos.ripple.com/repos/api/gpg/key/public" | gpg --dearmor > ripple-key.gpg
-        sudo mv ripple-key.gpg /usr/local/share/keyrings
+        sudo mv ripple-key.gpg /usr/local/share/keyrings/
```

Comment:
Hmm, as long as we're sudoing... let's cut out a step and just put it there in the first place? (I know I wrote this :expressionless:) 
```
wget -q -O - "https://repos.ripple.com/repos/api/gpg/key/public" | sudo gpg --dearmor -o /etc/apt/keyrings/ripple-key.gpg
```
Note the path change also: `/etc/apt/keyrings/`
Needs to be changed in 4. and 5. also.

Reply:
@legleux can you submit these changes in a new PR?